### PR TITLE
Minor code cleanup - removed unnecessary == true and other minor changes

### PR DESCRIPTION
**Pull request by SpadeInTheDirt**
so basically 

> i deleted all the == true 
> then i did it again for some reason 
> then i added the folder to itself somehow 
> then i removed it 
> then i removed some more == true that i missed and also refactored some and conditions to be separate 
       (ex. @cond (A and B) -> @cond A, @cond B
> then i added the folder to itself again somehow 
> and then i removed it 
> then i did some random piss change in bastion.opy

(basically i removed all the unnecessary == true, refactored some conditions and refactored some == false)|

organe

### CODE REVIEW

File: heroes/mercy.opy [hunk 1]
```diff
@@ -35,7 +35,8 @@ rule "Hero AI: Mercy Bot - Damaged Ally":
     @Hero mercy
     @Condition eventPlayer.Module_007_DummyBot_SeeDamagedA == true
     @Condition eventPlayer.CanSeeDeadAlly == false
-    @Condition (eventPlayer.CanSeeDeadAlly == false and eventPlayer.Mercy_Rez == true or eventPlayer.Mercy_Rez == false) == true
+    @Condition eventPlayer.CanSeeDeadAlly == false
+    @Condition (eventPlayer.Mercy_Rez or not eventPlayer.Mercy_Rez) == true
```
Comment (Zezombye) [inline heroes/mercy.opy]: "or" has less precedence than "and" smh

File: heroes/mercy.opy [hunk 2]
```diff
@@ -59,7 +60,8 @@ rule "Hero AI: Mercy Bot - Ally":
     @Hero mercy
     @Condition eventPlayer.Module_007_DummyBot_SeeDamagedA == false
     @Condition eventPlayer.Module_012_DummyBotAlly == true
-    @Condition (eventPlayer.CanSeeDeadAlly == false and eventPlayer.Mercy_Rez == true or eventPlayer.Mercy_Rez == false) == true
+    @Condition eventPlayer.CanSeeDeadAlly == false 
+    @Condition (eventPlayer.Mercy_Rez or not eventPlayer.Mercy_Rez) == true
```
Comment (Zezombye) [inline heroes/mercy.opy]: same

**Comment by Zezombye** [comment]
so ==true

File: heroes/mercy.opy (hunk 1 above)
Comment (SpadeInTheDirt) [reply heroes/mercy.opy]: oops

Comment (SpadeInTheDirt) [reply heroes/mercy.opy]: wait now how do i change it

Comment (SpadeInTheDirt) [reply heroes/mercy.opy]: oh nvm

**Pull request merged**